Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -224,20 +275,33 @@ static CFStringRef _CFStringFromLocalizedString(IDWriteLocalizedStrings* localiz
     // TODO::
     // Iterate through all attributed string ranges and identify attributes so they can be used to -
     //  - set indentation
-    //  - set font characteristics
+    //  - set kern
     //  - etc.
     //  These can be done using the DWrite IDWriteTextFormat range property methods.
     NSRange attributeRange;
     for (size_t i = 0; i < [ts->_attributedString length]; i += attributeRange.length) {
         NSDictionary* attribs = [ts->_attributedString attributesAtIndex:i
                                                    longestEffectiveRange:&attributeRange
                                                                  inRange:{ i, [ts->_attributedString length] - i }];
```

Comment:
This is being tested by CTLine_CreateTruncatedLineTruncationToken which makes sure the ranges are broken properly

---
In reply to: [79964738](https://github.com/Microsoft/WinObjC/pull/1021#discussion_r79964738) [](ancestors = 79964738)